mayhem/test_fuzz.mayhemfile

```diff
@@ -0,0 +1,4 @@
+project: nostril
```

Comment:
What is "test_fuzz"?

Reply:
I've renamed it as per your other suggestion